### DISCUSSION TITLE
Discharge Location SNF

### DISCUSSION THREAD

**Discussion by tomhilgers** [MIT-LCP/mimic-code#1306] [MIMIC-III] [answered]
Hi,
What does the discharge location "SNF" in the MIMIC-III data mean? I see it quite a lot but have never heard of it and cant find information about it online.

**Comment by marymlucas** [comment, marked as answer]
SNF usually stands for Skilled Nursing Facility.

**Reply by tomhilgers** [reply]
Thanks!

**Comment by Lauramoffat** [comment]
SNF stands for Skilled Nursing Facility
Laura

Sent from my iPhone

On May 20, 2022, at 7:09 AM, Tom Hilgers ***@***.***> wrote:

﻿

Hi,
What does the discharge location "SNF" in the MIMIC-III data mean? I see it quite a lot but have never heard of it and cant find information about it online.

**Reply by tomhilgers** [reply]
Thanks!
